samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -40,7 +40,7 @@ static K_SEM_DEFINE(sem_bis_sync_requested, 0U, 1U);
 static K_SEM_DEFINE(sem_bis_synced, 0U, CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT);
 
 /* Sample assumes that we only have a single Scan Delegator receive state */
-static const struct bt_bap_scan_delegator_recv_state *sink_recv_state;
+static const struct bt_bap_scan_delegator_recv_state *past_req_recv_state;
```

Comment:
The changes are good, but I think the `past` part of the name is misleading: Isn't this also used to sync with a broadcast source without using PAST? 

Suggest to just rename it `pa_req_recv_state` or `req_recv_state`

Reply:
Yes, I was also unsure about the proper name. Will change into req_recv_state